saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -143,49 +168,96 @@ msgstr "Administrateur de sauvegarde"
+#: internal_ops/forms.py:42 internal_ops/forms.py:172 internal_ops/forms.py:273
+#: internal_ops/forms.py:375
+msgid "Date Manager reviewed the request"
+msgstr "Date à laquelle le responsable a examiné la demande"
 
-#: internal_ops/forms.py:48 internal_ops/forms.py:181
+#: internal_ops/forms.py:43 internal_ops/forms.py:167 internal_ops/forms.py:274
+#: internal_ops/forms.py:376
+msgid "Manager Approved"
+msgstr "Gestionnaire approuvé"
+
+#: internal_ops/forms.py:44 internal_ops/forms.py:168 internal_ops/forms.py:275
+#: internal_ops/forms.py:377
+msgid "Manager Denied"
+msgstr "Gestionnaire refusé"
+
+#: internal_ops/forms.py:45 internal_ops/forms.py:173
 msgid "Submitted By"
 msgstr "Proposé par"
 
-#: internal_ops/forms.py:49 internal_ops/forms.py:182
+#: internal_ops/forms.py:46 internal_ops/forms.py:169 internal_ops/forms.py:174
+#: internal_ops/forms.py:276 internal_ops/forms.py:378
 msgid "Date Info Requested"
 msgstr "Informations sur la date demandées"
 
-#: internal_ops/forms.py:50 internal_ops/forms.py:183
+#: internal_ops/forms.py:47 internal_ops/forms.py:170 internal_ops/forms.py:175
+#: internal_ops/forms.py:277 internal_ops/forms.py:379
 msgid "Info Requested"
 msgstr "Informations demandées"
 
-#: internal_ops/forms.py:56 internal_ops/forms.py:184
+#: internal_ops/forms.py:48 internal_ops/forms.py:176 internal_ops/forms.py:279
+#: internal_ops/forms.py:381
 msgid "Fund Center"
 msgstr "Centre de fonds"
 
-#: internal_ops/forms.py:57 internal_ops/forms.py:185 templates/base.html:91
-#: templates/index.html:84
+#: internal_ops/forms.py:49 internal_ops/forms.py:177 internal_ops/forms.py:280
+#: internal_ops/forms.py:382 templates/base.html:91 templates/index.html:84
 msgid "S32 Approver"
 msgstr "Approbateur S32"
 
-#: internal_ops/forms.py:97 templates/internal_ops/view_request.html:77
+#: internal_ops/forms.py:84 templates/internal_ops/view_request.html:77
 msgid "Save"
 msgstr "Sauvegarder"
 
-#: internal_ops/forms.py:101
+#: internal_ops/forms.py:88
 msgid "Send for S32 Approval"
 msgstr "En attente d'envoi pour approbation S32"
 
-#: internal_ops/forms.py:105 internal_ops/forms.py:220
+#: internal_ops/forms.py:92 internal_ops/forms.py:212
 msgid "Request Additional Information"
 msgstr "Demander des informations supplémentaires"
 
-#: internal_ops/forms.py:115 internal_ops/forms.py:138
-#: internal_ops/forms.py:230 templates/internal_ops/view_request.html:47
+#: internal_ops/forms.py:102 internal_ops/forms.py:125
+#: internal_ops/forms.py:222 templates/internal_ops/view_request.html:47
 msgid "Record Purchase Information"
 msgstr "Enregistrer les informations d'achat"
 
-#: internal_ops/forms.py:367
+#: internal_ops/forms.py:171 internal_ops/forms.py:278
+#: internal_ops/forms.py:380
+msgid "Date Sent to S32 Approver"
+msgstr "Date d'envoi à l'approbateur S32"
+
+#: internal_ops/forms.py:281 internal_ops/forms.py:383
+msgid "S32 Approver approved"
+msgstr "Approbateur S32 approuvé"
```

Comment:
I think the syntax might be: "Approuvé par l'approbateur S32"

Although that does sound a bit odd...